COMPAT.md

```diff
@@ -45,6 +45,7 @@ This document describes the SQLite compatibility status of Limbo:
 | ON CONFLICT clause        | No      |         |
 | PRAGMA                    | Partial |         |
 | PRAGMA cache_size         | Yes     |         |
+| PRAGMA freelist_count     | Yes     |         |
```

Comment:
can we rebase with main and update the PRAGMA in another section?
in main we re-org the pragma statements a bit.